pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -266,12 +299,22 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
+            - mountPath: /var/lib/cephcsi/log/rbdDeployment
+              name: csi-log
           imagePullPolicy: {{ .ImagePullPolicy }}
         {{ end }}
       volumes:
         - name: host-dev
           hostPath:
             path: /dev
+        - name: csi-log
+          hostPath:
+            path: /var/lib/cephcsi/log/rbdDeployment
+            type: DirectoryOrCreate
+        - name: csi-logs-logrotate
+          hostPath:
+            path: /logrotate-config
+            type: DirectoryOrCreate
```

Comment:
Do we really need  a hostpath for it, isnt it possible to use some emptyDir? we cant use Hostpath `/logrotate-config` on host root dir

Reply:
We can do an empty directory, but having a host path seems more robust.
I did one thing I mounted it to /var/lib/cephcsi/logrotate-config

Reply:
Used empty dir